ansible_collections/arista/avd/plugins/plugin_utils/cv_client/workflows/deploy_cv_pathfinder_metadata_to_cv.py

```diff
@@ -84,6 +128,18 @@ def upsert_pathfinder(metadata: dict, device: CVDevice, studio_inputs: dict) ->
         "tags": {"query": f"device:{device.serial_number}"},
     }
 
+    pathfinder_location = {key: metadata.get(key) for key in ("region", "site", "address")}
+    if any(pathfinder_location):
+        if is_feature_supported("pathfinder_location", studio_version):
+            pathfinder_metadata.update(pathfinder_location)
```

Comment:
Maybe I am missing something but don't we need to do,
we need to do pathfinder_metadata["inputs"]["value"].update(..)

Reply:
fixed